packages/ember-engines/addon-test-support/index.js

```diff
@@ -54,6 +55,10 @@ async function loadEngine(engineName) {
     let assetLoader = owner.lookup('service:asset-loader');
 
     await assetLoader.loadBundle(engineName);
+
+    // ownerHasEngine does registry lookup and caches "miss".
+    // With register() we tell registry that module is now available.
+    owner.register(`engine:${engineName}`, require(`${engineName}/engine`).default);
```

Comment:
Why is it necessary? I didn't understand yet what was causing the issue.

Reply:
@villander I think this test messes up things https://github.com/villander/ember-engines/blob/master/packages/ember-engines/tests/acceptance/lazy-routeable-engine-test.js#L20

I'll try to see if we change that test setup instead and or use another API rather than `owner.hasRegistration`/`owner.factoryFor` to check if engine is available